src/main/java/tech/paramount/SnowflakeDialect.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package tech.paramount;
+
+import org.hibernate.dialect.DatabaseVersion;
+
+public class SnowflakeDialect extends org.hibernate.dialect.Dialect {
+
+    @Override
+    public DatabaseVersion getVersion() {
+        return new DatabaseVersion() {
+            @Override
+            public int getDatabaseMajorVersion() {
+                return 0;
+            }
+
+            @Override
+            public int getDatabaseMinorVersion() {
+                return 0;
+            }
+        };
+    }
+}
```

Comment:
Given that the `SnowflakeDialect` implementation here is basically a generalized shell of Hibernate's `Dialect` base class, I wonder whether it would make more sense to add a way that a user could influence the dialect chosen directly using the JDBC sink connector's dialect and perhaps we could omit this specific class all together somehow?